lib/apnotic/notification.rb

```diff
@@ -3,7 +3,8 @@
 module Apnotic
 
   class Notification < AbstractNotification
-    attr_accessor :alert, :badge, :sound, :content_available, :category, :custom_payload, :url_args, :mutable_content, :thread_id
+    attr_accessor :alert, :badge, :sound, :content_available, :category, :custom_payload, :url_args,
+                  :mutable_content, :thread_id, :type
```

Comment:
looking at the rest of the project, I think `type` might be a bit too generic.... what about `voip` as a boolean?

Reply:
from reading the apple documentation here: https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/sending_notification_requests_to_apns I think only one apns-push-type is allowed.  Due to that, I think we should ensure that we are not allowing more push type values to be set.  This is why I decided to go with `type`, and it could be set to the following values
- voip
- mdm
- fileprovider
- complication
- alert
-background

Since we are only concerned with `voip` we could limit the scope to just that, if we think that works better?